routers/api/packages/api.go

```diff
@@ -513,14 +513,23 @@ func CommonRoutes() *web.Route {
 			r.Get("/simple/{id}", pypi.PackageMetadata)
 		}, reqPackageAccess(perm.AccessModeRead))
 		r.Group("/rpm", func() {
+			// old rpm
```

Comment:
Only the old `GET` routes are needed. I suggest using a simple redirect with the default distribution. Then you don't need the `getDistribution` method.